server.py

```diff
@@ -159,6 +159,27 @@ def format_raw_course(raw_course):
 def schedule_sort_key(slot):
     return slot["days"], slot["startTime"], slot["endTime"]
 
+def time_slot_equal(slot_a, slot_b):
+    return slot_a["days"] == slot_b["days"] and \
+           slot_a["location"] == slot_b["location"] and \
+           slot_a["startTime"] == slot_b["startTime"] and \
+           slot_a["endTime"] == slot_b["endTime"]
```

Comment:
Why is this function necessary? Why can't we just compare `slot_a == slot_b`?

Reply:
That should work (I didn’t know that python has a deep compare for dict)

Reply:
That should work (I didn’t know that python has a deep compare for dict)

Reply:
I removed the method and now use normal equality check.